website/docs/language/import/generating-configuration.mdx

```diff
@@ -0,0 +1,149 @@
+---
+page_title: Import - Generating Configuration
+description: >-
+  Generate configuration and manage existing resources with Terraform using configuration-driven import.
+---
+
+# Generating Configuration
+
+~> **Experimental:** Configuration generation is available in Terraform v1.5 as an experimental feature. Later minor versions may contain changes to the formatting of generated configuration and behavior of the `terraform plan` command using the `-generate-config-out` flag.
+
+Terraform can generate HCL for resources that do not already exist in your configuration. Terraform produces HCL to act as a template that contains Terraform's best guess at the appropriate value for each resource argument. 
```

Comment:
Taking Rita's suggestion from down below to tweak this sentence to include "import":
```suggestion
Terraform can generate code for the resources it plans to import that do not already exist in your configuration. Terraform produces HCL to act as a template that contains Terraform's best guess at the appropriate value for each resource argument. 
```